### ISSUE TITLE
rst__expandButton and rst__collapseButton styling issue

### ISSUE THREAD

**Issue by Manali-addweb** [frontend-collective/react-sortable-tree#807]
I have been trying to change the styling of rst__expandButton and rst__collapseButton on the selected node or on some button not all on all the buttons. Any suggestion or any solution for this?

**Comment by kay-o** [comment]
possibly related? https://github.com/frontend-collective/react-sortable-tree/issues/661

**Comment by nahuelleiva** [comment]
What do you want to accomplish? Setting a certain style for a clicked node? If that's so, check #771 for reference.

**Comment by Manali-addweb** [comment]
@nahuelleiva No, I want to set style on each rst_node which expands/collapses icons by adding a custom class on that. I've checked reference but no luck found for my requirement. Actually, I want to indicate some symbols between tree steps using CSS.

**Comment by nahuelleiva** [comment]
@Manali-addweb can you provide an example? An image, or something that illustrates what you want more clearly.

**Comment by Manali-addweb** [comment]
@nahuelleiva  actually I want to do  some thing like this so for that want to add my custom class on each node .
![React-App4](https://user-images.githubusercontent.com/41608588/96574075-9cf7cf00-12ec-11eb-9b9a-4a4267ff848b.png)

**Comment by nahuelleiva** [comment]
So if I'm not getting this wrong, what you want to do is to build a relationship between nodes that are placed at a root level. And for this, you want to set like an icon (the icon with the vertical bars inside).
I do not know if this component is prepared to do that... But changing the classes will affect all the nodes at the same time whether you want it or not. You may want a custom class for that node in particular when you add it to the tree setting some conditions before.

Maybe you can try creating your own style class with all you need inside, and adding a property in the Node object in which you can store the value of the sibling node in the tree if it happens they are at the same level (in this case the root level), and only if you have a value in that property you'll set the style class to the node. It's an idea that I came up with, I cannot provide you a code example right now. Let me know if this works for you.

**Comment by Manali-addweb** [comment]
**Title**   adding custom class on node 
**Steps**  As per suggestion added property in the Node object and also added own class.
**What happens**  I Had added custom class on each node but it is not working . is there any way to add custom class on each node like we are adding  rowHeight  on   highlighted element  in image  
**OS version**  Ubuntu
![Uploading Screenshot-from-2020-10-21-16-24-21 (1).png…]()

**Comment by sinanyaman95** [comment]
Were you able to achieve this custom style? @Manali-addweb
